entities/snapshots/snapshot.go

```diff
@@ -17,9 +17,8 @@ type Snapshot struct {
 	StartedAt   time.Time `json:"startedAt"`
 	CompletedAt time.Time `json:"completedAt"`
 
-	ID       string   `json:"id"`
-	Files    []string `json:"files"`
-	BasePath string   `json:"basePath"`
```

Comment:
This `BasePath` is still needed for lower-level snapshotting mechanisms.

Reply:
We moved this to DataPath in the StorageProvider.  Are there situations where we will be working with a snapshot but not also a storage provider?

Reply:
I don't think so, base path should be removed then 🙂